tapalcatl_server/server.go

```diff
@@ -253,15 +253,28 @@ func (bm *OnDemandBufferManager) Get() *bytes.Buffer {
 func (bm *OnDemandBufferManager) Put(buf *bytes.Buffer) {
 }
 
+func logFatalCfgErr(logger JsonLogger, msg string, xs ...interface{}) {
+	logger.Error(LogCategory_ConfigError, msg, xs...)
+	os.Exit(1)
+}
+
 func main() {
 	var listen, healthcheck, debugHost string
 	var poolNumEntries, poolEntrySize int
 	var metricsStatsdAddr, metricsStatsdPrefix string
 
 	hc := handlerConfig{}
 
+	systemLogger := log.New(os.Stdout, "", log.LstdFlags|log.LUTC|log.Lmicroseconds)
+	hostname, err := os.Hostname()
```

Comment:
I tried running `hostname` on a machine, and it said:

```
tilequeue1
```

Should we be adding the stack information too, or is that added by cwlogs automatically? Would it make sense to use `os.Hostname()` as a default and allow it to be overridden by a command line argument or environment variable that we could set to `$hostname::$stackname` from runit / docker / whatever?